cantools/db/database.py

```diff
@@ -34,10 +34,10 @@ def __init__(self,
         self._version = version
         self._attribute_definitions = (attribute_definitions
                                        if attribute_definitions
-                                       else [])
+                                       else {})
```

Comment:
Leave this as a list as in DBC-format.

See https://github.com/eerimoq/cantools/blob/master/cantools/db/formats/dbc.py#L534-L541.

Reply:
Addressed in 34d11f81e538a5b8399cbb58477c67716b693977